### DISCUSSION TITLE
v0.8.3

### DISCUSSION THREAD

**Discussion by U-C-S** [U-C-S/Hurl#108] [Announcements]
Small update fixing various reported bugs and UX issues from `v0.8.2`

## Application Updates
- Installer now automatically installs .NET 8 before installing Hurl (#75)
- Clicking settings icon, now opens the `UserSettings.json` in default file handler for `JSON` files, instead of notepad (#89)
- Add a button for deleting current Timed default browser (#89)
- Support deleting rules (#105)
- Fix various crashes when opening Ruleset Manager, copying URL etc (#68, #102)
- Improve UX as per various reported issues (#89, #106)

With those fixed, Now I think it's time for improving the code quality of Hurl itself. Also, exploring to improve the startup times etc.

**Full Changelog**: https://github.com/U-C-S/Hurl/compare/v0.8.2...v0.8.3

<hr /><em>This discussion was created from the release <a href='https://github.com/U-C-S/Hurl/releases/tag/v0.8.3'>v0.8.3</a>.</em>

**Comment by stamminator** [comment]
Looks good! It's much more stable for me. On 0.8.1, I was frequently "losing" links by clicking into the background right at the moment Hurl's window appeared, causing it to close but not giving me a way to open it again. Not having the issue anymore on this release.
